supertokens_python/recipe/session/recipe_implementation.py

```diff
@@ -240,120 +350,331 @@ async def get_session(
     ) -> Optional[SessionContainer]:
         log_debug_message("getSession: Started")
 
-        log_debug_message(
-            "getSession: rid in header: %s", str(frontend_has_interceptor(request))
-        )
-        log_debug_message("getSession: request method: %s", request.method())
+        # This token isn't handled by getToken to limit the scope of this legacy/migration code
+        if request.get_cookie(LEGACY_ID_REFRESH_TOKEN_COOKIE_NAME) is not None:
+            # This could create a spike on refresh calls during the update of the backend SDK
+            return raise_try_refresh_token_exception(
+                "using legacy session, please call the refresh API"
```

Comment:
Please add a comment linking to the behaviour table of the function

Reply:
Done